modules/worker-utils/package.json

```diff
@@ -32,9 +32,10 @@
     "./src/lib/node/worker_threads.ts": "./src/lib/node/worker_threads-browser.ts",
     "./dist/es5/lib/node/worker_threads.js": "./dist/es5/lib/node/worker_threads-browser.js",
     "./dist/esm/lib/node/worker_threads.js": "./dist/esm/lib/node/worker_threads-browser.js",
-    "./src/lib/node/require-utils.node.ts": false,
-    "./dist/es5/lib/node/require-utils.node.js": false,
-    "./dist/esm/lib/node/require-utils.node.js": false,
+    "./src/lib/node/require-utils.node.ts": "./src/lib/node/require-utils.browser.ts",
+    "./dist/lib/node/require-utils.node.js": "./dist/lib/node/require-utils.browser.js",
```

Comment:
Is there actually a root level dist? I think 3.4 always builgs two dists es5/esm? 

it won't hurt (but could confuse)